LoadTest/Models/Common/OperationResultResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿using DigitalOffice.LoadTesting.Models.Enums;
+using System.Collections.Generic;
+
+namespace DigitalOffice.LoadTesting.Models.Responses.Templates
+{
+    public class OperationResultResponse<T>
```

Comment:
can you use the class from kernel?

Reply:
i don't include kernel to this project, because I don't need it